dockers/docker-database/database_config.json.j2

```diff
@@ -52,6 +58,11 @@
             "id" : 7,
             "separator": "|",
             "instance" : "redis"
+        },
+        "GLOBAL_APP_DB" : {
+            "id" : 8,
```

Comment:
since this is a new redis instance, should it start from 0?

Reply:
There is an issue on handling multi-instances in [table.cpp](https://github.com/Azure/sonic-swss-common/blob/master/common/table.cpp),  const TableNameSeparatorMap TableBase::tableNameSeparatorMap doesn't consider multi-instance case, I opened a new issue to track it - [5351](https://github.com/Azure/sonic-buildimage/issues/5351)